src/Pyz/Yves/CheckoutPage/Theme/default/components/molecules/checkout-list/checkout-list.twig

```diff
@@ -22,6 +22,7 @@
                     required: false,
                     attributes: {
                         checked: choice.value == data.checkedValue,
+                        'aria-label': 'checkout_page.aria_label.shipping_method' | trans,
```

Comment:
why shipping? we use this component in payments. Just move aria,label to the choice property, and mark it from parent component